composer.json

```diff
@@ -67,11 +68,21 @@
 
     "scripts": {
         "lint": [
-            "Composer\\Config::disableProcessTimeout"
+            "Composer\\Config::disableProcessTimeout",
+            "vendor/bin/php-cs-fixer fix --dry-run --diff --show-progress=dots"
         ],
         "test": [
             "Composer\\Config::disableProcessTimeout",
             "vendor/bin/phpunit ./tests"
+        ],
+        "fix": [
```

Comment:
But I guess it's the same as other commands though, right? I mean, it is supposed to fix anything that might need fixing... e.g. later on even calling `phpstan --fix` if that becomes available to non-pro usage. I wouldn't be too worried about it since devs can always just check what has changed with git.

Do you still feel strongly about it?

Reply:
Ah, OK - if the intention is it fixes anything / everything I think I'm OK with that.